src/main/java/net/sf/jabref/MetaData.java

```diff
@@ -30,9 +30,10 @@
 
     public static final String META_FLAG = "jabref-meta: ";
     public static final String SAVE_ORDER_CONFIG = "saveOrderConfig";
+    public static final String SAVE_ACTIONS = "saveActions";
```

Comment:
But nonetheless the cleanup actions would be only invoked when the database is saved. So saveActions is also appropriate. If you want I can change it to the common denominator `saveCleanupActions` ?